api/src/test/java/org/openmrs/LocationTest.java

```diff
@@ -36,8 +36,8 @@ public void isInHierarchy_shouldShouldFindLocationInHierarchy() {
 		locationGrandParent.addChildLocation(locationParent);
 		locationParent.addChildLocation(locationChild);
 		
-		assertTrue(Location.isInHierarchy(locationChild, locationParent));
-		assertTrue(Location.isInHierarchy(locationChild, locationGrandParent));
+		Assertions.assertTrue(Location.isInHierarchy(locationChild, locationParent));
```

Comment:
can you please use static imports for the assertions

Reply:
Please try my settings using the `Migrate to Junit 5` on the single class here. That for me gives a nice diff with static imports.